doc/tutorials/filesets.rst

```diff
@@ -262,9 +262,9 @@ This includes changes such as:
 
 * Adding SOP instances using the :meth:`FileSet.add()
   <pydicom.fileset.FileSet.add>` or :meth:`FileSet.add_custom()
-  <pydicom.fileset.FileSet.add_custom>` methods
+  <pydicom.fileset.FileSet.add_custom>` methods.
```

Comment:
I think no period is needed in such lists, if the list item is not a sentence (I'm not a native speaker, so I may be wrong).

Reply:
I tend to agree, in bullet or point form lists usually a period is not needed

Reply:
@mrbean-bremen @darcymason that's correct (according to the internet); however, the style should be consistent within a single list ([ref](https://proofed.com/writing-tips/should-bullet-point-lists-end-with-a-period/)). I added these periods here to conform with the next two items.

Reply:
In this case I suggest to remove the periods in thjese two items, otherwise this just looks wrong to me.